src/components/charts/SummaryChartGroup/index.jsx

```diff
@@ -6,6 +6,59 @@ import './SummaryChartGroup.css';
 import helper from '../helper.js';
 
 class SummaryChartGroup extends Component {
+  getChart(item) {
+    switch (item.type) {
+    case 'pie':
+      return (
+        <SummaryPieChart
+          data={item.data}
+          title={item.title}
+          lockValue={this.props.lockValue}
+          lockMessage={this.props.lockMessage}
+          useCustomizedColorMap={this.props.useCustomizedColorMap}
+          customizedColorMap={this.props.customizedColorMap}
+          maximumDisplayItem={this.props.maximumDisplayItem}
+          chartIsEmpty={item.chartIsEmpty}
+          chartEmptyMessage={this.props.chartEmptyMessage}
+        />
+      );
+    case 'full-pie':
+      return (
+        <SummaryPieChart
+          data={item.data}
+          title={item.title}
+          lockValue={this.props.lockValue}
+          lockMessage={this.props.lockMessage}
+          useCustomizedColorMap={this.props.useCustomizedColorMap}
+          customizedColorMap={this.props.customizedColorMap}
+          maximumDisplayItem={this.props.maximumDisplayItem}
+          chartIsEmpty={item.chartIsEmpty}
+          chartEmptyMessage={this.props.chartEmptyMessage}
+          innerRadius={0}
+        />
+      );
+    case 'bar':
+      return (
+        <SummaryHorizontalBarChart
+          data={item.data}
+          title={item.title}
+          vertical
+          color={this.props.useCustomizedColorMap ? undefined
+            : this.props.barChartColor}
+          lockValue={this.props.lockValue}
+          lockMessage={this.props.lockMessage}
+          useCustomizedColorMap={this.props.useCustomizedColorMap}
+          customizedColorMap={this.props.customizedColorMap}
+          maximumDisplayItem={this.props.maximumDisplayItem}
+          chartIsEmpty={item.chartIsEmpty}
+          chartEmptyMessage={this.props.chartEmptyMessage}
+        />
+      );
+    default:
```

Comment:
I think this switch statement doesn't account for the stackedBar chart type, now that the default is to return null

Reply:
Yeah this is expected. The `stackedBar` chart actually belongs to its own UI component and will not be put into the `SummaryChartGroup`. So in here we don't expect to see `stackedBar`

See this part of code for reference https://github.com/uc-cdis/data-portal/blob/master/src/GuppyDataExplorer/ExplorerVisualization/index.jsx#L57-L61

Reply:
ah very good